.github/workflows/release.yml

```diff
@@ -28,7 +30,38 @@ on:
             - 'false'
           default: 'false'
 jobs:
+  should-build-change:
+    runs-on: ubuntu-latest
+    outputs:
+        repo-cache-hit: ${{ steps.should-build-change.outputs.REPO_CACHE_HIT }}
```

Comment:
So `should-build-change` (job) has an "output" called `repo-cache-hit` which is set to the value of the `REPO_CACHE_HIT` (env) set in the `should-build-change` (step). It should be possible to get rid of one "chain" here.

What if the output for `should-build-change` (job) was just set to `steps.cache-last-commit.outputs.cache-hit == 'true'` ?